packages/x-data-grid-premium/tsconfig.build.json

```diff
@@ -15,6 +15,10 @@
     { "path": "../x-data-grid-pro/tsconfig.build.json" },
     { "path": "../x-license/tsconfig.build.json" }
   ],
-  "include": ["src/**/*.ts*", "../../node_modules/@mui/material/themeCssVarsAugmentation"],
+  "include": [
+    "src/**/*.ts*",
+    "./node_modules/@mui/material/themeCssVarsAugmentation",
+    "./node_modules/@mui/material/**/*.d.ts"
```

Comment:
Should be in `/// <reference types="@mui/types" />` instead?

Reply:
I've updated the solution: https://github.com/mui/mui-x/pull/11875/files#diff-1e719331900b3b401f52ede41edfac6b60464e1e2faf36ece511cff7cd21a99f
We no longer need any strange explicit type declarations in this case.
I had to update https://github.com/mui/mui-x/pull/11875#pullrequestreview-2014640878 `typescript` to avoid `tsc` package resolution issues.
WDYT?